tests/service/FsUnit.fs

```diff
@@ -1,23 +1,23 @@
 ﻿module FsUnit
 
 open System.Diagnostics
-open NUnit.Framework
-open NUnit.Framework.Constraints
+open Xunit
 
 [<DebuggerNonUserCode>]
-let should (f : 'a -> #Constraint) x (y : obj) =
```

Comment:
This all is not needed anymore really - Xunit checks same types by default in its assertions.
`should` is only used with `equal` now.

I will likely remove this completely, don't want to do it here to not explode the diff.